src/Home/ProximityTracingActivationStatus.tsx

```diff
@@ -15,13 +14,10 @@ export const ProximityTracingActivationStatus: FunctionComponent = () => {
   const { applicationName } = useApplicationName()
   const { t } = useTranslation()
 
-  const {
-    authorized: isAuthorized,
-    authorizedAndEnabled: isProximityTracingOn,
-  } = exposureNotifications.status
+  const { status } = exposureNotifications
 
   const handleOnPressFix = () => {
-    if (!isAuthorized && isPlatformiOS()) {
```

Comment:
do we need to keep the isPlatformiOS check here?

Reply:
We were thinking of that, but I don't think so? If the concept of having to "authorize" ENs is specific to iOS, the status should never be UNAUTHORIZED on android. Can you think of a reason that might be the case?